doc/lua_api.txt

```diff
@@ -2725,6 +2726,37 @@ Some types may inherit styles from parent types.
     * noclip - boolean, set to true to allow the element to exceed formspec bounds. Default to false.
 * label, vertlabel
     * noclip - boolean, set to true to allow the element to exceed formspec bounds.
+    * horigin - Sets horizontal origin in relation to label text. Possible values:
+        * `left`: Left side of the bounding box
+        * `center`: Center of the bounding box
+        * `right`: Right side of the bounding box
+        * `leftline`: **Note**: Only for negative `newline_spacing`
```

Comment:
I don't understand this comment?

Reply:
This was meant to help remove confusion by showing that `left` acts the same as `leftline` (same applies to `top` and `topline` for `vorigin`) when `newline_spacing` is positive, since the left side of the whole rect and the left side of the first line are exactly the same.  But it apparently adds more confusion; I can just remove it, unless you have a better way of putting it.